.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Same comment, why push this directory ?

Reply:
C'estun pr draft samuel, ne perd pas de temp avec ^^

Reply:
quand se sera bon pour être review jenlèverrai le draft